Projects/Application/Sources/Common/Extensions/TimeInterval/Double+TimeInterval.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Created by Maxim Berezhnoy on 2019-02-12.
+// Licensed under the MIT license
+//
+//
```

Comment:
Extra comment line in all headers.

Reply:
Damn. I will fix my template and the files.

Reply:
FIxed